libtelemetry/src/test/java/com/mapbox/android/telemetry/TelemetryClientAppUserTurnstileEventTest.java

```diff
@@ -9,16 +14,19 @@
 
 import okhttp3.Callback;
 
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 public class TelemetryClientAppUserTurnstileEventTest extends MockWebServerTest {
 
   @Test
   public void sendsTheCorrectBodyPostingAppUserTurnstileEvent() throws Exception {
+    initializeMapboxTelemetry();
```

Comment:
Why do we need to initialize `MapboxTelemetry`?
We're using `MapboxTelemetry.applicationContext` when retrieving the vendor id from `SharedPreferences` (`TelemetryUtils`) so setting a mocked context should be enough 👀

```java
Context mockedContext = mock(Context.class, RETURNS_DEEP_STUBS);
MapboxTelemetry.applicationContext = mockedContext;
```